Editor/ModelImporter_GLTF.cpp

```diff
@@ -3006,3 +3170,1238 @@ void Import_Mixamo_Bone(LoaderState& state, Entity armatureEntity, Entity boneEn
 		get_humanoid().bones[size_t(HumanoidComponent::HumanoidBone::RightToes)] = boneEntity;
 	}
 }
+
+template<typename T>
+inline void _ExportHelper_valuetobuf(const T& input, tinygltf::Buffer& buffer_builder, size_t& buf_i)
+{
+	const size_t _right = buf_i + sizeof(input);
+	if (_right > buffer_builder.data.size())
+	{
+		buffer_builder.data.resize(_right);
+	}
+	*(T*)(buffer_builder.data.data() + buf_i) = input;
+	buf_i = _right;
+}
+
+inline tinygltf::Value _ExportHelper_tovalue(float input)
+{
+	return tinygltf::Value(input);
+}
+
+inline tinygltf::Value _ExportHelper_tovalue(XMFLOAT3 input)
+{
+	auto value_builder = tinygltf::Value(tinygltf::Value::Array({
+		tinygltf::Value(input.x), tinygltf::Value(input.y), tinygltf::Value(input.z)
+	}));
+	return value_builder;
+}
+
+inline tinygltf::Value _ExportHelper_tovalue(XMFLOAT4 input)
+{
+	auto value_builder = tinygltf::Value(tinygltf::Value::Array({
+		tinygltf::Value(input.x), tinygltf::Value(input.y), tinygltf::Value(input.z), tinygltf::Value(input.w)
+	}));
+	return value_builder;
+}
+
+wi::vector<std::string> original_texture_extension_iterator = 
+{
+	"png",
+	"jpg",
+	"jpeg",
+};
+
+inline std::string _ExportHelper_GetOriginalTexture(std::string texture_file)
+{
+	for(auto& ext : original_texture_extension_iterator)
+	{
+		std::string target_file = wi::helper::ReplaceExtension(texture_file, ext);
+		wi::backlog::post(target_file);
+		if(std::filesystem::exists(target_file))
```

Comment:
Should use `wi::helper::FileExists()` instead, because uwp platform can't use std::filesystem

Reply:
About filesystem, is it ok to extend the wicked's filesystem functions to cover all functionalities from std::filesystem?

Reply:
No, we don't need to write a complete file system replacement. It can be done on a case-by case basis when you need a feature that must be supported by all platforms.